src/Microsoft.Extensions.CodeGenerators.Mvc/View/ViewGenerator.cs

```diff
@@ -85,11 +85,11 @@ public virtual IEnumerable<string> TemplateFolders
             Contract.Assert(model != null, "Validation succeded but model type not set");
 
             var appbasePath = _applicationEnvironment.ApplicationBasePath;
-            var outputPath = Path.Combine(
-                appbasePath,
-                Constants.ViewsFolderName,
-                model.Name,
-                viewGeneratorModel.ViewName + Constants.ViewExtension);
+            var outputFolder = String.IsNullOrEmpty(viewGeneratorModel.RelativeFolderPath)
+                ? Path.Combine(appbasePath)
+                : Path.Combine(appbasePath, viewGeneratorModel.RelativeFolderPath);
+
+            var outputPath = Path.Combine(outputFolder, viewGeneratorModel.ViewName + Constants.ViewExtension);
```

Comment:
Like I mentioned before, I intentionally removed the convention from command line ... for example, in VS, if user chose context menu on project and created a view using Add View dialog, the relative path would be empty and the expected behavior is to generate view in the project folder but not inside Views folder ... hence I needed to remove the convention from command line ... So if someone wants to generate a view inside 'Views' folder from command line, they need to pass '-outdir Views' explicitly...